ExampleMod/build.txt

```diff
@@ -1,5 +1,6 @@
 author = bluemagic123
 version = 0.9.1
+languageVersion = 6
```

Comment:
I'll agree, as much as I personally love C# 6, and use it in tModLoader itself, we'll use the old way in ExampleMod. I'll request you just use the longhand property overrides